frontend/src/lib/components/LemonSelect.tsx

```diff
@@ -134,6 +131,7 @@ export function LemonSelect<O extends LemonSelectOptions>({
                     maxContentWidth: dropdownMaxContentWidth,
                 }}
                 icon={localValue && allOptions[localValue]?.icon}
+                // so that the pop up isn't shown along with the close button
                 sideIcon={isClearButtonShown ? <div /> : undefined}
```

Comment:
I think there is a generally better fix by somehow merging LemonButtonWithSideAction and LemonButtonWithPopup... I would need to try it myself but I think that would solve all these issues...

Reply:
Oh, yep... Am just taking some small steps while I figure out how this all interacts :)